Makefile

```
@@ -372,12 +371,19 @@ build-test-app:
 deploy-test-app:
 	@make --directory='./src/acceptance/assets/app/go_app' deploy
 
+.PHONY: build-acceptance-test-app
+build-acceptance-test-app:
```

The name confuses me, why `-app`?

```suggestion
.PHONY: build-acceptance-tests
build-acceptance-tests:
```